alacritty_terminal/src/ansi.rs

```diff
@@ -338,9 +338,22 @@ pub enum CursorStyle {
     Hidden,
 }
 
+impl Default for CursorShape {
+    fn default() -> CursorShape {
+        CursorShape::Block
+    }
+}
+
+/// Describes style of cursor (shape + blinking).
```

Comment:
This comment doesn't really provide any value. `Describes the style of cursor` is the same as `CursorStyle` and the comment of `shape + blinking` is just all the fields. We shouldn't document fields because those might change, so that's actually actively harmful. I think it would be better to just delete this rather than using it as-is.

Reply:
We can keep it at `Describes the style of the cursor.` I don't really care either way. But I wouldn't remove existing documentation for this.